src/pineko/evolve.py

```diff
@@ -11,11 +11,21 @@
 import rich.box
 import rich.panel
 import yaml
-from eko.io.types import ScaleVariationsMethod
+from eko.io.types import EvolutionMethod, ScaleVariationsMethod
 
 from . import check, comparator, ekompatibility, version
 
 
+def axes(grid, max_as=None, max_al=None):
```

Comment:
I agree that it makes sense to have the two cases together, but since it doesn't make much sense to have the max for one and not the other (and indeed in that case you're ignoring the passed one), I'd collect them in a single argument:
```suggestion
def axes(grid, maxs: Optional[Tuple[int]]=None):
```

Reply:
[2cbedfb](https://github.com/NNPDF/pineko/pull/75/commits/2cbedfb06f19413cc18d5b86c41170287267a1da)